ceph_bootstrap/config_shell.py

```diff
@@ -311,6 +363,7 @@ def possible_values(self):
                 =========================================
                 Options to control the deployment of Ceph and other services
                 ''',
+        'advanced': True,
         'options': {
             'Bootstrap': {
                 'type': 'flag',
```

Comment:
The `'help'` string for `'Bootstrap'` option should say `cephadm bootstrap` instead of `ceph-daemon --bootstrap`

Reply:
https://github.com/SUSE/ceph-bootstrap/pull/36